### DISCUSSION TITLE
How to masking the TextInputColumn ?

### DISCUSSION THREAD

**Discussion by altimusin** [filamentphp/filament#4996] [Help]
Hi All,

when using TextInput component-form, we can put masking,.
```
use Filament\Forms\Components\TextInput;
 
TextInput::make('name')
    ->mask(fn (TextInput\Mask $mask) => $mask->pattern('+{7}(000)000-00-00'))
```
But how masking the Text input column ?

Thanks

**Comment by altimusin** [comment]
Anyone can help me ?.
Thank You

**Comment by altimusin** [comment]
Anyone can help me?

How to mask the TextColumn ?

Thanks

**Reply by danharrin** [reply]
You can use this - https://filamentphp.com/docs/2.x/tables/columns/text#custom-formatting

But you'll need to do the formatting of the string yourself in PHP
